docs/design/chart_format.md

```diff
@@ -189,25 +176,14 @@ keywords:
 * proxy
 source: https://github.com/foo/bar
 home: http://nginx.com
-depends:
-        charts:
-          - name: memcached
-                location: helm.sh/charts/memcached
-                version: 0.9.3
-        kubernetes:
-                version: “>=1.0.0”
-                extensions:
-                  - extensions/v1beta1
+expander:
+  name: GoTemplates
```

Comment:
They won't work for several reasons including that.  Literally the only thing that actually works right now is DM-based charts.  When the go templates expander is integrated, in order to work in the current implementation, they need to be changed to return DM types instead of K8s resources.  I believe then they will work.  That change is one of the items in #520